scripts/install/etc/sysconfig/motr

```diff
@@ -151,3 +151,6 @@ MOTR_TRACED_KEEP_LOGS_NUM=2
 # for developers to quickly test their local changes w/o creating and installing
 # an rpm package each time.
 #MOTR_DEVEL_WORKDIR_PATH=/home/devvm/motr
+
+#MOTR ADDB Record Max record size in bytes (10G)
+MOTR_MOD_ADDB_RECORD_SIZE=10737418240
```

Comment:
So the default ADDB_SIZE is the max value?
Is it too large?

Reply:
Kept it to 1G